File: Braintree/src/main/java/com/braintreepayments/api/exceptions/ConfigurationException.java
package com.braintreepayments.api.exceptions;

/**
 * Error class thrown when a configuration value is invalid
 */
public class ConfigurationException extends BraintreeException {
    
    public ConfigurationException(String message, Throwable cause) {
        super(message, cause);
    }
    
    public ConfigurationException(String message) {
        super(message);
    }
}

File: BraintreeCore/src/main/java/com/braintreepayments/api/core/ConfigurationException.kt
package com.braintreepayments.api.core

import androidx.annotation.RestrictTo

/**
 * Error class thrown when a configuration value is invalid
 */
class ConfigurationException @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) constructor(
    message: String?,
    cause: Throwable? = null
) : BraintreeException(message, cause)
